symfony-cmf/cmf-phpcr-dbal-pack/1.0/config/packages/cmf_phpcr_dbal.yaml

```diff
@@ -0,0 +1,24 @@
+framework:
+    templating: { engines: ['twig'] }
+
+cmf_menu:
+    persistence:
+        phpcr: ~
+
+cmf_routing:
+    chain:
+        routers_by_id:
+            cmf_routing.dynamic_router: 200
+            router.default:             100
+    dynamic:
+        persistence:
+            phpcr:
+                enabled: true
+
+cmf_content:
+    persistence:
+        phpcr: ~
+
+cmf_core:
```

Comment:
Why can't you put this is a symfony-cmf/core-bundle recipe?

Reply:
the idea is to have one pack for all the cmf things. is that the wrong approach?

should we have packs for each bundle and then a meta-pack that requires all the other packs?

Reply:
One pack to rule them all... Are the bundles to tightly coupled together? Say that I only want to install cmf-core, shouldn't I get some help from a recipe?

Reply:
it would be easier, when all other bundles would require `core-bundle` as a base container, which serves the dependencies. But that isn't. And hiding the the dependencies and the recipe in a package isn't that clean i think.

Reply:
most bundles depend on core-bundle being present, and that is modeled in composer.json - but from that, CmfCoreBundle is not instantiated in the kernel. afaik those bundles do not require services from core-bundle however, just php code.

the core-bundle handles sharing some configuration that is relevant for all cmf bundles through prepend. 

thus, separate packs would work. i just feel often you want all the main cmf bundles to have a more complete thing. can packs depend on other packs? then we could have separate packs for each bundle and this pack bring them all together.